trigger/wsclient/trigger.go

```diff
@@ -91,6 +167,95 @@ func (t *Trigger) Start() error {
 
 // Stop stops the trigger
 func (t *Trigger) Stop() error {
+	err := t.wsconn.WriteMessage(websocket.CloseMessage, []byte("Closing connection while stopping trigger"))
+	if err != nil{
+		t.logger.Infof("Error received: [%s] while sending close message when Stopping Trigger", err)
```

Comment:
error

Reply:
converted to warn level as its not critical to be logged as error. done.